code/src/App.js

```diff
@@ -1,9 +1,29 @@
-import React from 'react'
+import React, { useState } from "react";
+import Form from "Form.js";
+import Answers from "Answers.js"
```

Comment:
Tip is to add these to a `Components` folder so it will be easier to navigate if you add 50 more files to the project later on